server/services/notificationService.js

```diff
@@ -13,52 +20,94 @@ const createNotificationService = emailClient => {
       .add(3, 'days')
       .format('dddd D MMMM')
 
-  const sendReminder = async (emailAddress, { reporterName, incidentDate }) =>
+  const sendReporterStatementReminder = async (emailAddress, { reporterName, incidentDate }) =>
     emailClient
-      .sendEmail(config.email.templates.involvedStaff.REMINDER, emailAddress, {
+      .sendEmail(reporter.REMINDER, emailAddress, {
         personalisation: {
           REPORTER_NAME: reporterName,
           INCIDENT_DATE: asDate(incidentDate),
           INCIDENT_TIME: asTime(incidentDate),
           DEADLINE_DATE: asDeadline(incidentDate),
+          LINK: emailUrl,
+        },
+        reference: null,
+      })
+      .then(({ body }) => logger.info(`Send statement reminder, successful for reporter: '${reporterName}'`, body))
+      .catch(({ message }) => logger.error(`Send statement reminder, failed for reporter: '${reporterName}'`, message))
```

Comment:
is it right that we don't throw exception on failure to send email?
Is the logger monitored for failures or would this better to be a custom event in analytics?
How about throwing exception and then setting the next reminder to be the following day instead?

Reply:
We do have a story to add events to analytics - as we will want to track both successes and failures of this. 

I'll have a think about the error handling - I mostly didn't want to block the queue if an error sending emails happens - though as you say, catching the error higher up and giving another chance to send the 'overdue' email might be a good move